src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.jsx

```diff
@@ -15,20 +15,21 @@ import IndicatorStore from 'app/stores/indicatorStore';
 import {joinTeam, leaveTeam} from 'app/actionCreators/teams';
 import LoadingError from 'app/components/loadingError';
 import LoadingIndicator from 'app/components/loadingIndicator';
-import OrganizationState from 'app/mixins/organizationState';
 import {Panel, PanelHeader} from 'app/components/panels';
 import InlineSvg from 'app/components/inlineSvg';
 import EmptyMessage from 'app/views/settings/components/emptyMessage';
 import {t} from 'app/locale';
 import space from 'app/styles/space';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
+import withOrganization from 'app/utils/withOrganization';
+import SentryTypes from 'app/sentryTypes';
 
 const TeamMembers = createReactClass({
   displayName: 'TeamMembers',
   propTypes: {
-    api: PropTypes.object,
+    api: PropTypes.object.isRequired,
+    organization: SentryTypes.Organization.isRequired,
   },
-  mixins: [OrganizationState],
```

Comment:
@lynnagara With mixins removed from this component, this may be an opportunity to convert this component into a class component and remove `createReactClass` from this file.

Reply:
It’d be easier to have a new pr that codemods the entire codebase to remove createreactclass imo